src/pow/cuckaroo/cuckaroo.h

```diff
@@ -25,11 +25,9 @@
 #define EDGEBITS 29
 #endif
 
-#ifndef CYCLELEN
+//#ifndef CYCLELEN
 // the next most important parameter is the (even) length
 // of the cycle to be found. a minimum of 12 is recommended
```

Comment:
These can also be moved. Or move the last two lines to where the `CYCLELEN` is defined.